tasks/Task.cpp

```diff
@@ -93,13 +93,17 @@ bool Task::configureHook()
     cVb.buildFrom(cMb);
     task.set_cVe(cVb);
 
+    //set the time for trigger the timeout
+    timeout = base::Timeout(base::Time::fromSeconds(1));
```

Comment:
is TIMEOUT an exception on the controllers ? I thought it was a runtime error...

I think it makes sense to have this component port driven on the image.